### ISSUE TITLE
Undesirable scrollbar after pictures adding

### ISSUE THREAD

**Issue by mbarbetti** [alshedivat/al-folio#356]
### Discussed in https://github.com/alshedivat/al-folio/discussions/328

<div type='discussions-op-text'>

<sup>Originally posted by **mbarbetti** June 30, 2021</sup>
Adding pictures to `news` announcements, a horizontal scrollbar appears in the same section, regardless of the size of the pictures. How can I fix it?
![highlights](https://user-images.githubusercontent.com/55536905/123977648-792e7200-d9bf-11eb-89c0-d93f5d40ac7e.png)</div>

**Comment by alshedivat** [comment]
@mbarbetti, thanks for raising this issue. Do you mind posting a minimal code snippet that would help reproducing it?

**Comment by mbarbetti** [comment]
@alshedivat for sure!

In the [_config.yml](https://github.com/alshedivat/al-folio/blob/master/_config.yml) file I have set `news_limit` to 3. Then I have added several announcements to the [_news](https://github.com/alshedivat/al-folio/tree/master/_news) folder, and the last three are the following:

**1.**
```Markdown
---
layout: post
date: 2021-07-31 15:00:00-0000
inline: true
---

Test 1 (no pictures insertion)
```

**2.**
```Markdown
---
layout: post
date: 2021-07-31 16:00:00-0000
inline: true
---

Test 2 (pictures insertion)

<div class="row mt-3 align-items-center justify-content-sm-center">
    <div class="col-sm-6 mt-3 mt-md-0">
        <img class="img-fluid rounded z-depth-1" src="{{ site.baseurl }}/assets/img/4.jpg">
    </div>
    <div class="col-sm-3 mt-3 mt-md-0">
        <img class="img-fluid rounded z-depth-1" src="{{ site.baseurl }}/assets/img/11.jpg">
    </div>
</div>
```

**3.**
```Markdown
---
layout: post
date: 2021-07-31 17:00:00-0000
inline: true
---

Test 3 (no pictures insertion)
```

This is what I obtain:
![screen](https://user-images.githubusercontent.com/55536905/127747251-c6ce9215-10ff-48f2-954a-cbe5ad0de927.png)

**Comment by mbarbetti** [comment]
@alshedivat, have you any news to fix this issue (#356)?
[sorry for the spam but @stale worried me a bit...]
